cmd/token.go

```diff
@@ -101,11 +108,37 @@ The OIDC token includes claims regarding the GCE VM, which is verified by Attest
 		}
 		gceAK.Close()
 
+		var cloudLogClient *logging.Client
+		var cloudLogger *logging.Logger
+		if cloudLog {
+			if audience == "" {
+				return errors.New("cloud logging requires the --audience flag")
```

Comment:
capitalize Cloud Logging, as it's a proper noun

Reply:
go-staticcheck would give "error strings should not be capitalized (ST1005)"